test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] As it is said, this file should be ignored by git, this means that it shouldn't appear here in your repo, removing it, you can use `git rm -f test.md` and writing it on gitignore, you should remove the starting slash, and then commit changes